index.html

```diff
@@ -0,0 +1,419 @@
+        <p>What would become our observable universe mushroomed in size at least a trillion trillionfold — from a submicroscopic speck of primordial energy to the size of a grapefruit — in less than a cosmic eye-blink.</p>
+        <p>Almost as quickly, this pulse would subside, relaxing into ordinary particles and radiation. All of normal cosmic history was still ahead, resulting in today’s observable universe, a patch of sky and stars billions of light-years across. “It’s often said that there is no such thing as a free lunch,” Dr. Guth likes to say, “but the universe might be the ultimate free lunch.”</p>
+        <p>Make that free lunches. Most of the hundred or so models resulting from Dr. Guth’s original vision suggest that inflation, once started, is eternal. Even as our own universe settled down to a comfortable homey expansion, the rest of the cosmos will continue blowing up, spinning off other bubbles endlessly, a concept known as the multiverse.</p>
+        <p>So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into the past.</p>
+        <p>We might never know what happened before inflation, at the very beginning, because inflation erases everything that came before it. All the chaos and randomness of the primordial moment are swept away, forever out of our view.</p>
+        <p>“If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”</p>
+      </article>
+
+      <div id="article-foot">
+        <p class="foot-note">A version of this article appears in print on March 17, 2014, Section A, Page 1 of the New York edition with the headline: Space Ripples Reveal Big Bang’s Smoking Gun. Order Reprints | Today’s Paper | Subscribe</p>
+        <div class="container">
+          <button class="blue-btn">READ 615 COMMENTS</button>
+          <div class="social-media">
+            <i class="fab fa-facebook-f"></i>
+            <i class="fab fa-twitter"></i>
+            <i class="fa fa-envelope"></i>
+            <i class="fa fa-share"></i>
+          </div>
+        </div>
+      </div>
+    </main>
+
+    <!--News article ending-->
+    <!--
+    <aside>
+      <div id="sign-up">
+        <h2>Sign up for Science Times</h2>
+        <p>We’ll bring you stories that capture the wonders of the human body, nature and the cosmos.</p>
+        <button class="blue-btn">sign up</button>
+      </div>
+      <div id="editors-picks-aside">
+        <h2>Editors’ Picks</h2>
+        <img src="img/bob-ross-cover.png" alt="">
+        <a href="#">Where Are All the Bob Ross Paintings? We Found Them.</a>
+        <img src="img/merlin.jpg" alt="">
+        <a href="#">Need Etiquette Tips for Cannabis? For Starters, Don’t Call It ‘Marijuana’ or ‘Weed’</a>
+        <img src="img/snapshot.jpg" alt="">
+        <a href="#">When a Dating Dare Leads to Months of Soul Searching</a>
+      </div>
+    </aside>
+-->
+
```

Comment:
You have these two sections commented, which should be on the right side of the website.